### PR TITLE
Create Syntax.md

### PR DESCRIPTION
**Pull request by Denchick**

### CODE REVIEW

File: docs/Syntax.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+psql (14.5 (Homebrew), server console)
+Type "help" to get help.
+
+news=> SHOW segments;
+list of data segments
+---------------------------
+data card with the shard01 identifier data card
+with the shard02 identifier
+(2 rows)
```
Comment (EinKrebs) [inline docs/Syntax.md]: There's no such method, appararently

File: docs/Syntax.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+To make all things work, the router needs to know the following:
+
+- What tables are you query
```
Comment (EinKrebs) [inline docs/Syntax.md:28]:
```suggestion
- What tables do you query
```

File: docs/Syntax.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+- Which columns in each table should the router look
```
Comment (EinKrebs) [inline docs/Syntax.md:29]: look on/look for

File: docs/Syntax.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+- The type of these columns
```
Comment (EinKrebs) [inline docs/Syntax.md:30]:
```suggestion
- Types of these columns
```

File: docs/Syntax.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+- Mapping from [range of values] to [shard number]
+
+Let's create a distribution first:
```
Comment (EinKrebs) [inline docs/Syntax.md:33]: Maybe it's better to describe various sharidng-related terms before showing an example?

File: docs/Syntax.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+```
+➜ psql "host=localhost sslmode=disable user=news dbname=news port=7432"
+demo=> CREATE A ds1 DISTRIBUTION OF integer COLUMN TYPES;
+add a distribution kit
+----------------------------------
+created distribution with the ds1 identifier
+(1 line)
```
Comment (EinKrebs) [inline docs/Syntax.md:41]:
```suggestion
demo=> CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
         add distribution         
----------------------------------
 created distribution with id ds1
(1 row)
```

File: docs/Syntax.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+The next step is to specify a list of tables and columns. TODO how to make it smaller
```
Comment (EinKrebs) [inline docs/Syntax.md]:
```suggestion
The next step is to specify a list of tables and columns. Making it simpler is TBD
```

File: docs/Syntax.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+And at the end specify a list of ranges: which values to route to which shard. Note: The right bound is infinity if there are no key ranges.
+
+```
+CREATE A krid1 KEY RANGE FROM ROUTE 1 TO sh1 FOR ds1 ALLOCATION;
+add a range of keys
+--------------------------------
+a key range has been created with a binding of 1
+(1 line)
```
Comment (EinKrebs) [inline docs/Syntax.md:67]:
```suggestion
CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1 FOR DISTRIBUTION ds1;
         add key range          
--------------------------------
 created key range with bound 1
(1 row)
```

File: docs/Syntax.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+CREATE A krid2 KEY RANGE FROM ROUTE 11 TO sh1 FOR ds1 ALLOCATION;
+add a range of keys
+---------------------------------
+a key range has been created with a binding of 11
+(1 row)
```
Comment (EinKrebs) [inline docs/Syntax.md:73]:
```suggestion
CREATE KEY RANGE krid2 FROM 11 ROUTE TO sh1 FOR DISTRIBUTION ds1;
          add key range          
---------------------------------
 created key range with bound 11
(1 row)
```

File: docs/Syntax.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+CREATE A krid3 KEY RANGE FROM ROUTE 21 TO sh2 FOR ds1 ALLOCATION;
+add a range of keys
+---------------------------------
+a range of keys has been created with a binding of 21
+(1 line)
```
Comment (EinKrebs) [inline docs/Syntax.md:79]:
```suggestion
CREATE KEY RANGE krid3 FROM 21 ROUTE TO sh2 FOR DISTRIBUTION ds1;
          add key range          
---------------------------------
 created key range with bound 21
(1 row)
```

**Review by EinKrebs — CHANGES_REQUESTED** [review]

File: docs/Syntax.md (hunk 7 above)
Comment (Denchick) [reply docs/Syntax.md]: just drop this todo

**Review by x4m — APPROVED** [review]

**Pull request merged**